_includes/code/automated-testing.py

```diff
@@ -0,0 +1,126 @@
+# Blog: Automated testing for Weaviate applications - Python code
+# START
+import weaviate
+import os
+import ijson
+import csv
+
+MAX_ROWS_TO_IMPORT = 50  # TODO set to something like 50 to limit vectorization API calls
+
+# Instantiate the embedded Weaviate client with the user/password and OpenAI API key
+client = weaviate.Client(
+    embedded_options=weaviate.embedded.EmbeddedOptions(),
+    additional_headers={
+        'X-OpenAI-Api-Key': os.environ['OPENAI_API_KEY']  # Replace w/ your OpenAI API key
+    }
+)
+
+# ============================
+# ===== Define the class =====
+# ============================
+class_name = 'JeopardyQuestion'
+class_definition = {
+    'class': class_name,
+    'description': 'A Jeopardy! question',
+    'vectorizer': 'text2vec-openai',
+}
+
+# Clean slate for local testing (GitHub Actions VMs start fresh) because Weaviate data is persisted in embedded mode.
+if client.schema.exists(class_name):
+    client.schema.delete_class(class_name)
+if not client.schema.exists(class_name):
+    client.schema.create_class(class_definition)
```

Comment:
This feels more complicated that it needs to be.

Maybe we can just replace it with:
```
# Clean slate for local testing (GitHub Actions VMs start fresh) because Weaviate data is persisted in embedded mode.
client.schema.delete_all()
```

Then later you could create a new collection, and even do a little assert to verify the collection is there.

Yes, I know that this would be testing Weaviate functionality, but it would be good to start with a small win.
To show how in a few lines of code we can create a new collection and then `assert` that the collection is there.
(in a way, this could be testing if the collection configuration is valid, as one day we might introduce breaking changes and then this test would flag it)